<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMaster</class>
 <widget class="QMainWindow" name="MMaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор матриц</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="printerBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Матрица</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Масштаб матрицы: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="scale">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>; Размер матрицы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minus">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plus">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="go">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Анализ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Создать матрицу с размером:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Матрица из файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Матрица в файл</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="randBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Заполнить рандомно</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Заполнить выбранную ячеку: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zeroBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oneBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delBTN">
        <property name="font">
         <font>
          <family>Montserrat Alternates</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить матрицу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="TVLayout">
      <item row="1" column="1">
       <widget class="QGraphicsView" name="telek">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
